test/units/galaxy/test_token.py

```diff
@@ -32,6 +34,47 @@ def b_token_file(request, tmp_path_factory):
         C.GALAXY_TOKEN_PATH = orig_token_path
 
 
+def test_client_id(monkeypatch):
+    monkeypatch.setattr(C, 'GALAXY_SERVER_LIST', ['server1', 'server2'])
+
+    test_server_config = {option[0]: None for option in SERVER_DEF}
+    test_server_config.update(
+        {
+            'url': 'http://my_galaxy_ng:8000/api/automation-hub/',
+            'auth_url': 'http://my_keycloak:8080/auth/realms/myco/protocol/openid-connect/token',
+            'client_id': 'galaxy-ng',
+            'token': 'access_token',
+        }
```

Comment:
should this also be added to the docs as an example (as listed in https://github.com/ansible/ansible/issues/75593)?

Reply:
@samccann Good catch, thanks! I added an example.